src/lsst/cst/conversions/tools.py

```diff
@@ -1,27 +1,21 @@
-"""data science format utils."""
-
-from dataclasses import dataclass
+"""Tools for common conversions."""
 
 import numpy as np
-import pandas as pd
+import json
+import warnings
 
-from lsst.cst.utilities.image import CalExpId
+from lsst import geom
```

Comment:
lsst.geom is not available out of the lsst stack so cannot be used. I do a try...catch ImportError around this naughty imports.

Reply:
Ah! Thank you. I found an example of try/except with ImportError in another .py file and copied it.